landing-pages/site/content/en/blog/airflow-survey-2020/index.md

```diff
@@ -0,0 +1,426 @@
+
+**When onboarding new members to airflow, what is the biggest problem?**
+
+|                                                                 |   No. |     % |
+|-----------------------------------------------------------------|-------|-------|
+| No guide on best practises on developing DAGs                   |   102 | 50.25 |
+| There is no easy option to launch Airflow                       |    64 | 31.53 |
+| Small number of tutorials on different aspects of using Airflow |    57 | 28.08 |
+| Documentation is not clear enough                               |    53 | 26.11 |
+| There is no easy option to deploy DAGs to an Airflow instance   |    52 | 25.62 |
+| No problems                                                     |    34 | 16.75 |
+| Small number of blogs regarding Airflow                         |    30 | 14.78 |
+
+**Which interface(s) of Airflow do you use as part of your current role?**
+
+|                                                       |   No. |     % |
+|-------------------------------------------------------|-------|-------|
+| Original Airflow Graphical User Interface             |   199 | 98.03 |
+| CLI                                                   |    88 | 43.35 |
+| API                                                   |    48 | 23.65 |
+| Custom (own created) Airflow Graphical User Interface |    12 |  5.91 |
+| Other                                                 |     3 |  1.48 |
+
+**Do you combine multiple DAGs?**
+
+|                                    |   No. |     % |
+|------------------------------------|-------|-------|
+| Yes, by triggering another DAG     |    87 | 42.86 |
+| No, I don't combine multiple DAGs  |    79 | 38.92 |
+| Yes, through SubDAG                |    40 | 19.7  |
+| Other                              |    18 |  8.87 |
+
+
+**How do you integrate with external services?**
+
+|                                             |   No. |     % |
+|---------------------------------------------|-------|-------|
+| Using existing dedicated operators / hooks  |   147 | 72.41 |
+| Using Bash / Python operator                |   140 | 68.97 |
+| Using own custom operators / hooks          |   138 | 67.98 |
+| Other                                       |    12 |  5.91 |
+
+**What external services do you use in your Airflow DAGs?**
+
+|                                                   |   No. |     % |
+|---------------------------------------------------|-------|-------|
+| Amazon Web Services                               |   121 | 59.61 |
+| Internal company systems                          |   113 | 55.67 |
+| Google Cloud Platform / Google APIs               |    97 | 47.78 |
+| Hadoop / Spark / Flink / Other Apache software    |    72 | 35.47 |
+| Microsoft Azure                                   |    21 | 10.34 |
+| Other                                             |    19 |  9.36 |
+| I do not use external services in my Airflow DAGs |     5 |  2.46 |
```

Comment:
These percentage don't add up to 100 - were users allowed to choose more than one option? Adding more detail about the percentage may help folks to make sense of the numbers.

Reply:
Good point, I will add to question line information about single/multi choice 👌